### PR TITLE
Make clear that Liberapay is legal

### PR DESCRIPTION
**Pull request by MartinDelille**

### CODE REVIEW

File: www/about/faq.spt [hunk 1]
```diff
@@ -55,6 +55,10 @@ title = _("Frequently Asked Questions")
             '</a>'|safe) }}</p>
     </dd>
 
+    {{ dt(_("Can Liberapay be used to laundry money?"), 'laundry-money') }}
```
Comment (Changaco) [inline www/about/faq.spt]: :D that's a funny way of phrasing it, but nobody would ever ask "is it okay if I use your service to do something illegal?" as the answer is obviously "no".

Comment (MartinDelille) [inline www/about/faq.spt]: Glad you like it! ;-)

Comment (MartinDelille) [inline www/about/faq.spt]: This is a first draft. Any suggestion is appreciated, but what do you think in making it clear in the FAQ?

Comment (Changaco) [inline www/about/faq.spt]: What we want to do is reassure (potential) users about the legality of Liberapay, so I think the question would be more like "Is Liberapay legal?" or "Does Liberapay respect financial regulations?".

File: www/about/legal.spt [hunk 2]
```diff
@@ -21,7 +21,8 @@ title = _("Legal")
 <p>{{ _(
     "This website is managed by Liberapay, a non-profit organization founded in 2015 and legally registered in France.\n"
     "Its publication director is Liberapay's legal representative, {0}.",
-    "Charly Coste"
+    "Charly Coste\n"
+    "Liberapay respects EU financial regulations."
```
Comment (Changaco) [inline www/about/legal.spt]: That's not a valid way to add a sentence to this paragraph.

**Comment by MartinDelille** [comment]
I fixed the paragraph issue.

**Comment by Changaco** [comment]
Squashed and improved.

I know what information I want to put on the Legal page, but I don't actually have it, so I've sent a request to MangoPay.

**Comment by MartinDelille** [comment]
It would be nice if they could directly interact on Github...

**Comment by Changaco** [comment]
I never received a response from MangoPay, I'm going to send them another email, and in the meantime I'm merging this as is.

**Comment by Changaco** [comment]
MangoPay has sent me proof that Liberapay is indeed registered as an "e-money distributor" at the [ACPR](https://acpr.banque-france.fr/). I'm not seeing a registration number that we could display on the website, so we'll just say that we're registered.

**Pull request merged**